modules/installation-aws-config-yaml.adoc

```diff
@@ -87,35 +88,39 @@ platform:
       adminContact: jdoe
       costCenter: 7536
 ifdef::vpc[]
-    subnets: <6>
+    subnets: <7>
     - subnet-1
     - subnet-2
     - subnet-3
 endif::vpc[]
 pullSecret: '{"auths": ...}' <1>
 ifdef::vpc[]
-fips: false <7>
-sshKey: ssh-ed25519 AAAA... <8>
+fips: false <8>
+sshKey: ssh-ed25519 AAAA... <9>
 endif::vpc[]
 ifndef::vpc[]
-fips: false <6>
-sshKey: ssh-ed25519 AAAA... <7>
+fips: false <7>
+sshKey: ssh-ed25519 AAAA... <8>
 endif::vpc[]
 ifdef::private[]
-publish: Internal <9>
+publish: Internal <10>
 endif::private[]
 ----
 <1> Required. The installation program prompts you for this value.
-<2> If you do not provide these parameters and values, the installation program
+<2> Optional. Add this parameter to force the Cloud Credential Operator
```

Comment:
We're using _Optional:_

Reply:
Oh! didn't realize that was for callouts too. Thanks :)